cfgov/unprocessed/css/molecules/multi-select.less

```diff
@@ -93,7 +88,12 @@
             margin-right: unit(15px / @base-font-size-px, em);
         }
 
+        li:last-child {
+          margin-bottom: unit(5px / @base-font-size-px, em);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/3f3c9ab1a5e29762a3a6e40a717b86629b192c39